### DISCUSSION TITLE
Kibana user management

### DISCUSSION THREAD

**Discussion by yula-21** [Security-Onion-Solutions/securityonion#6898] [Unsupported Versions] [answered]
Hi,
Im trying to manage users in my kibana on Security Onion (version 2.3.50)

Ive created new users but it seems that when I’m trying to logout from the default user (which is named **anonymous**) and log in again with my user the system automatically logs me in as **anonymous**

Does anyone have any idea how to solve this issue?

**Comment by RobertClum** [comment]
Did you try clearing cookies on your browser > close browser > reopen browser, then try?

**Comment by dougburks** [comment, marked as answer]
Security Onion 2.3.50 is quite old. Please update to the latest version:
https://docs.securityonion.net/en/2.3/soup.html

Then see:
https://docs.securityonion.net/en/2.3/so-elastic-auth.html
